service/viper_service.go

```
@@ -0,0 +1,51 @@
+/* This service focus on reading the Environment Files of config.yaml using viper package.  */
+
+package service
+
+import (
+	"log"
+
+	"github.com/spf13/viper"
+)
+
+//SetupViper: Make the configuration for the viper module (config file, define paths, read config.yaml file).
+func SetupViper() {
+	//Set config filename
+	viper.SetConfigName("config")
+	viper.SetConfigType("yaml")
+	//Set config file path.
+	viper.AddConfigPath(".")    //Normal Server Running.
+	viper.AddConfigPath("./..") //In case of Testing Go files.
+
+	err := viper.ReadInConfig()
+	if err != nil {
+		log.Fatal("Error: Couldn't Read or find the Environment File.")
+	}
+}
+
+//GetEnvVariable: Find a String value inside config.yaml(Environment variable file).
+func GetEnvVariable(key string) string {
+	SetupViper()
+
+	//This method returns value (string) and a boolean
+	value, ok := viper.Get(key).(string)
+	if !ok {
+		log.Fatalf("Error: Couldn't find the string value(%v) on the Environment Variable file.", key)
```

If a variable is not found, we can assume the variable is expecting this value won't have a valid init value; if this is the case, I can also consider your application won't be working as expected, right?

Being so, I'd also return an error in this function, and halt program execution if an error is returned by this method.

The same comment applies to the GetIntEnvVariable method.